src/c4/test/tstifpstream.cc

```diff
@@ -0,0 +1,105 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/test/tstifpstream.cc
+ * \author Mathew Cleveland
+ * \date   Feb. 2020
+ * \brief  Tests ifpstream class for paralle read
+ * \note   Copyright (C) 2016-2020 Triad National Security, LLC.
```

Comment:
* fix dates
* Any assumptions about this test?  Should it be disabled for scalar builds?

Reply:
This should still work in scalar mode.